Adding mission statement

hannafer opened the pull request.

hannafer committed:
Adding mission statement

hannafer committed:
Adding ND logo

hannafer committed:
Adding a smaller ND logo

hannafer committed:
Adding ND logo

hannafer committed:
Deleting the smaller logo

hannafer committed:
Merge branch 'readme_updates' of https://github.com/kyduff/qcnd into readme_updates

hannafer committed:
Changing size of the ND logo

hannafer committed:
Update README.md

hannafer committed:
Update README.md

kyduff commented:
This looks good so far. We'll need more content and the official logo before merging.

kyduff committed:
Tweak format and add git content.

* Try sections without numbers
* Try hyperlinking some resources
* Try bulleting
* Add a "why we use git" section

kyduff committed:
Hyperlink everything and add installation instructions.

kyduff committed:
Fix formatting issue in instructions

kyduff committed:
Add emojis to instructions.

kyduff committed:
Change emojis to look better.

The gift emoji looked pretty bad. I replaced with checkmarks because they look better and make it seem more like a todo list.

hannafer committed:
Cosmetic changes, adding Club Communication section

kyduff committed:
Hyperlink slack invitation

hannafer committed:
Adding communication links

kyduff committed:
Reformat and add content

* Add "Leadership" section and place at bottom
* Add paragraph in about section elaborating on club activities
* Fill content for section about how to get involved
* Add information about Belvedere JHT info session
* Change case of section headers
* Add officer titles

kyduff committed:
Update slack link

hannafer committed:
Table of contents links
